docs/README.md

```diff
@@ -1,10 +1,15 @@
+
+# Table of Contents
+
 Here is a table of contents to help you find the document you need:
 
 ## Template creation and management details
 
 * [How to create templates guide](template-creation-how-to-guide.md)
 * [Partial template conceptual guide](partial-template-concept.md)
 * [Examples of using helper functions](using-helpers-concept.md)
+* [HL7 v2 template creation](template-creation-guide-HL7v2.md)
+* [CCDA template creation](template-creation-guide-CCDA.md)
```

Comment:
Should this have the hyphen in C-CDA?

Reply:
@moiradillon12 , Should we be using CDA or CCDA?

Reply:
I have been saying CDA when I refer to CDA documents but C-CDA everywhere else. @ranvijaykumar do you know if it should be CDA template or C-CDA template?